impeller/renderer/backend/vulkan/capabilities_vk.cc

```diff
@@ -330,12 +343,14 @@ bool CapabilitiesVK::SupportsFramebufferFetch() const {
 
 // |Capabilities|
 bool CapabilitiesVK::SupportsCompute() const {
-  return false;
+  // Vulkan 1.1 requires support for compute.
```

Comment:
oh, this will work?

Reply:
It's not wired up yet. But capabilities wise it is supported.

Reply:
I think we'll start crashing on drawPoints on Vulkan if this returns true before its completely ready?

Reply:
Ahh, yes. I'm not sure how big of a deal that is. I could make this patch still return false here but true on supports subgroups for now...?

Reply:
sg

Reply:
(We'll hit a codepath that ends up doing an IMPELLER_UNIMPLEMENTED which will abort)